doc/sign_requests.md

```diff
@@ -0,0 +1,142 @@
+[box-sign-request-signer]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxSignRequestSigner.html
+[box-sign-request-file]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxSignRequestFile.html
+
+Get all Sign Requests
+------------------------
+
+Calling the static [`getAll(BoxAPIConnection api, String... fields)`][get-all-sign-requests]
+will return an iterable that will page through all the sign requests.
+It is possible to specify limit of items per single
+response and fields to retrieve by calling the static
+[`getAll(int limit, BoxAPIConnection api, String... fields)`][get-all-sign-requests-with-fields]
+method.
+
+<!-- sample get_all_sign_requests -->
+```java
+Iterable<BoxSignRequest.Info> signRequests = BoxSignRequest.getAll(api);
+for (BoxSignRequest.Info signRequestInfo : signRequests) {
+	// Do something with the signRequest.
+}
+```
+
+[get-all-sign-requests]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxSignRequest.html#getAll-com.box.sdk.BoxAPIConnection-java.lang.String...-
+[get-all-sign-requests-with-fields]: http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxSignRequest.html#getAll-int-com.box.sdk.BoxAPIConnection-java.lang.String...-
+
+Get Sign request by ID
+------------------------
+
+Calling [`getInfo(String fields ...)`][get-sign-request-by-id] will return [`BoxSignRequest.Info`][box-sign-request-info] object
+containing information about the sign request. If necessary to retrieve
+limited set of fields. It is possible to specify them using param.
+
+<!-- sample get_sign_request_by_id -->
+```java
+BoxSignRequest signRequest = new BoxSignRequest(api, id);
+BoxSignRequest.Info signRequestInfo = signRequest.getInfo();
+```
+
+[get-sign-request-by-id]:http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxSignRequest.html#getInfo-java.lang.String...-
+[box-sign-request-info]:http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxSignRequest.Info.html
+
+
+Cancel Sign request
+------------------------
+
+Calling [`cancel()`][cancel-sign-request] will cancel ongoing sign request.
+
+<!-- sample cancel_sign_request -->
+```java
+BoxSignRequest signRequest = new BoxSignRequest(api, id);
+BoxSignRequest.Info signRequestInfo = signRequest.getInfo();
+
+signRequestInfo.cancel();
+```
+
+[cancel-sign-request]:http://opensource.box.com/box-java-sdk/javadoc/com/box/sdk/BoxSignRequest.html#cancel--
+
+
+Resend Sign request
+------------------------
+
+Calling [`resend()`][resend-sign-request] will resend a sign request to all signers that have not signed it yet.
+There is an about 10-minute cooling-off period between sending reminder emails. If this method is called during the
+cooling-off period, [`BoxAPIException`][box-api-exception] will be thrown.
+
+<!-- sample resend_sign_request -->
+```java
+BoxSignRequest signRequest = new BoxSignRequest(api, id);
+BoxSignRequest.Info signRequestInfo = signRequest.getInfo();
+
```

Comment:
probably can be omitted

Reply:
Let's keep it, for the reason above (dev docs).